packages/minifier-js/.npm/package/npm-shrinkwrap.json

```diff
@@ -7,14 +7,14 @@
       "integrity": "sha512-b0553uYA5YAEGgyYIGYROzKQ7X5RAqedkfjiZxwi0kL1g3bOaBNNZfYkzt/CL0umgD5wc9Jec2FbB98CjkMRvQ=="
     },
     "source-map": {
-      "version": "0.5.7",
-      "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.5.7.tgz",
-      "integrity": "sha1-igOdLRAh0i0eoUyA2OpGi6LvP8w="
+      "version": "0.6.1",
```

Comment:
We haven't taken this minor version update in other places, but I think the usage of `source-map` by `uglify-es` is fairly self-contained.